model/ring.go

```diff
@@ -19,10 +19,11 @@ type Ring struct {
 	Version            string
 	State              string
 	Provisioner        string
+	ChangeRequest      *ChangeRequest
```

Comment:
I wonder if it would not be better to keep this data on `Ring` level, something like:
```
DesiredImage
DesiredVersion
```

Going even step further, maybe a good model would be to separate `Ring` from `RingRelease` all together, this however could be a much bigger undertaking, and I did not thought it through that deep, so just something to consider. 

The `Ring` could then be simplified, and the `RingRelease` would store all data such as `Image`, `Version`, `ReleaseStart` etc. We could even store some more data such as release duration and have a historical view of those, perhaps even rolling back etc.